kafka-rest/pom.xml

```diff
@@ -67,6 +67,12 @@
             <artifactId>slf4j-log4j12</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+        <!-- Use a repackaged version of log4j with security patches. Default log4j v1.2 is a transitive dependency of slf4j-log4j12, but it is excluded in common/pom.xml -->
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>confluent-log4j</artifactId>
+            <version>${confluent-log4j.version}</version>
```

Comment:
does this need this or would version just be inherited?

Reply:
not strictly necessary. But doesn't hurt either. I just followed what every other dependency block had in this file.